README.md

```diff
@@ -269,6 +269,34 @@ Function arguments are:
     building the layer. This is mainly useful to ignore the
     configuration file from the container layer.
 
+### `nix2container.shadowLayer`
```

Comment:
I think it would be nice to mention the example because i'm not sure it's easy to get how to use this helper.